### ISSUE TITLE
why use EmbeddingLoss?

### ISSUE THREAD

**Issue by james20141606** [magenta/ddsp#53]
Hi, I have a question about embedding loss used in autoencoder, more specifically, `PretrainedCREPEEmbeddingLoss` used in ae_abs.gin. From the codes it seems to regularize the original and reconstruction audio's latent f0, right? I am not quite sure why you do this, is it kind of like the cycle loss? since you calculate the latent again from the reconstructed audio?
I am not sure what this loss is used for, is it for training the CREPE model?  I just found the logic is a little weird here.
Thank you!

**Comment by jesseengel** [comment]
Hi, it's not actually training the CREPE model, it's using the difference
in activations of a pretrained model as a loss function that can take into
account higher order statistical dependencies than say L1 or L2 of a
spectrogram. This is a direct analogy of "perceptual loss functions" in
images and style transfer literature (ex.
https://deepai.org/machine-learning-glossary-and-terms/perceptual-loss-function
)

**Comment by james20141606** [comment]
thanks for the explanation!
